src/Components/Comments.jsx

```diff
@@ -19,9 +19,9 @@ function Comments() {
                 setIsLoading(false)
          });}
          }, [article_id])
-
-         console.log(comments[0])
-         return isLoading ? (<p>Loading!</p>) : ( 
+        
+         console.log(comments)
+         return isLoading ? (<p>Loading!</p>) : comments === undefined ? (<p>No Comments for this Article </p>) : ( 
             <div>  
     <h3>Comments for Article {`${comments[0].article_id}`}</h3>
```

Comment:
This line should be changed, because if there are no comments the whole page breaks - let's try keep it flexible!

<img width="529" alt="image" src="https://user-images.githubusercontent.com/99129306/206435470-d57277b5-0209-47dc-9a57-102cc0df8948.png">